python/pyiceberg/table/snapshots.py

```diff
@@ -59,34 +58,19 @@ class Summary(IcebergBaseModel):
     like snapshot expiration, to skip processing certain snapshots.
     """
 
-    __root__: Dict[str, Union[str, Operation]]
+    operation: Operation = Field()
     _additional_properties: Dict[str, str] = PrivateAttr()
 
-    @root_validator
```

Comment:
A lot of changes here because the additional fields in the operation `dict` are considered properties. Since the `@root_validator` is not there anymore, we completely had to rewrite this. For the better I'd say.